test/TestCases/Expressions/Abs.cpp

```diff
@@ -22,7 +22,7 @@
 
 #include <vctr_test_utils/vctr_test_common.h>
 
-TEMPLATE_PRODUCT_TEST_CASE ("Abs", "[abs]", (PlatformVectorOps, VCTR_NATIVE_SIMD), (float, double, int32_t, int64_t, std::complex<float>, std::complex<double>, std::complex<int32_t>, std::complex<int64_t>) )
```

Comment:
why remove?

Reply:
Mhm, the diff looks like this, not sure why GitHub does not display the additions..?
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/13800135/221878467-40e5733b-2c96-48d2-a512-9156dfc2a20a.png">

Reply:
In case your question was why I removed the `std::complex<int>` types: I had to find out that `std::complex` is only defined for floating point types and causes build errors on some platforms if used with integer types.